reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -92,26 +93,79 @@ class StaticHtmlReporter : Reporter {
                     }
                 }
 
+                fun getTitleExt(mData: Map<String, String>): String {
+                    return if (mData.isNotEmpty()) arrayOf(mData["SW_NAME"],
+                        mData["SW_VERSION"]).joinToString(separator = " ") else ""
+                }
+
                 div {
                     id = "report-container"
 
+                    val titleExtStr = getTitleExt(reportTableModel.metadata)
+
                     div("ort-report-label") {
                         +"Scan Report"
+                        +(if (titleExtStr != "") " for " else "")
```

Comment:
What do you think about moving the logic from above line into `getTitleExt` so that this part of the code becomes a bit more `boring` to read. You could then also eliminate the `titleExtStr` variable which imo would also need a better name if it wasn't removed.

Reply:
Done